addons/ondevice-controls/src/types/Boolean.tsx

```diff
@@ -3,40 +3,24 @@ import React from 'react';
 
 export interface BooleanProps {
   onChange: (value: boolean) => void;
-  knob: {
+  arg: {
     name: string;
     value: boolean;
   };
 }
 
-class BooleanType extends React.Component<BooleanProps, {}> {
-  static defaultProps = {
-    knob: {},
-    onChange: (value) => value,
-  };
-
-  static serialize = (value) => (value ? String(value) : null);
-
-  static deserialize = (value) => value === 'true';
-
-  onValueChange = () => {
-    const { onChange, knob } = this.props;
-    onChange(!knob.value);
-  };
-
-  render() {
-    const { knob } = this.props;
-
-    return (
-      <View style={styles.switch}>
-        <Switch testID={knob.name} onValueChange={this.onValueChange} value={knob.value} />
-      </View>
-    );
-  }
-}
+const BooleanType = ({ arg, onChange = (value) => value }: BooleanProps) => (
+  <View style={styles.switch}>
+    <Switch testID={arg.name} onValueChange={() => onChange(!arg.value)} value={arg.value} />
```

Comment:
Do we have any use for the `testID` - `arg.name` -combo? Or should we just remove both? Simply carried them over from the old implementation, but if we're not using them for anything then it would make sense to just get rid of them.

Reply:
Well I think having a testID might be useful for people wanting to do automated testing on top of storybook. However I'm not sure if anyone does use it for that.

To me it's fine if you remove it since I expect to add unit tests later on which will introduce testID's in a more structured way I think.

Reply:
Good point: It's possible that someone is using this. Again, not a big deal either way, but keeping it as-is might have some value. Left it for now, let's reconsider as appropriate.

Reply:
sounds good.